sdk/eventgrid/eventgrid/src/eventGridClient.ts

```diff
@@ -32,6 +33,16 @@ export type EventGridPublisherClientOptions = CommonClientOptions;
  */
 export type SendOptions = OperationOptions;
 
+/***
+ * Options for the send events operation, when the input schema is cloud event.
+ */
+export type CloudEventSendOptions = SendOptions & {
```

Comment:
would it make sense to instead declare this like

```ts
export interface CloudEventSendOptions extends SendOptions {

}
```

Reply:
Makes sense to me!